src/business-logic/services/tv2-inews-ingest-service.ts

```diff
@@ -0,0 +1,12 @@
+import { IngestService } from './interfaces/ingest-service'
+import {HttpRequestParameters, HttpService} from './interfaces/http-service'
+
+export class Tv2InewsIngestService implements IngestService {
+  constructor(private readonly httpService: HttpService) {}
+  public async reloadIngestData(rundownName: string): Promise<void> {
```

Comment:
Be nice with a newline between each method. That includes the constructor.

Reply:
This has been changed!